pom.xml

```diff
@@ -413,6 +413,29 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
```

Comment:
Why did you add all those profiles? The main build seems to be fast enough even with all this enabled.

Reply:
Ah, when i initially try to do something with the parser I like to have the DEBUG_PARSER and DEBUG_LOOKAHEAD options set to true. When you run `mvn clean package` with those, the build takes quite a while (mostly from all the printing). I can revert these since they're only just for my own benefit when trying to add new things, and it's mostly cause i set all the debug options!

Reply:
No if you find them useful keep them, but keep the standard behaviour untouched.

Reply:
cool, updated!